data-serving/data-service/src/controllers/case.ts

```diff
@@ -88,40 +98,43 @@ const dtoFromCase = async (storedCase: CaseDocument) => {
         });
     }
 
-    if (ageRange && creator) {
```

Comment:
This was the main culprit causing issues with tests. this if SHOULD only contain age range and NOT the creator. Latter was added because of eslint which required check for a variable that based on the api requirements MUST be present, but due to the other issue in the tests was not. Decoupling this if into two separate ones resolves the issue.